CMakeLists.txt

```diff
@@ -345,60 +345,62 @@ set(STRIP_TABLE_CMD "awk '/Problem Summary/ { while (!match($0, /termination/)){
 # If running in a BSUB allocaation, use jrun commands and make sure we are
 # requesting the correct resources.
 if(HIOP_TEST_WITH_BSUB)
-  set(RUNCMD "jsrun -n 1")
+  set(RUNCMD "jsrun" "-n" "1")
   set(MPICMD "jsrun")
   if(HIOP_USE_GPU)
-    set(MPICMD "${MPICMD} -g 1")
-    set(RUNCMD "${RUNCMD} -g 1")
+    set(MPICMD ${MPICMD} "-g" "1")
+    set(RUNCMD ${RUNCMD} "-g" "1")
   endif()
 else()
-  set(RUNCMD "") # No special command is needed to run this program
+  set(RUNCMD "mpirun" "-n" "1")
```

Comment:
I think you'll have to check if `HIOP_USE_MPI` is on before using mpirun.

Reply:
Just added a commit to address this. Should be fixed now